pandas/tests/formats/test_format.py

```diff
@@ -3004,6 +3004,66 @@ def test_to_latex_multiindex(self):
 
         self.assertEqual(result, expected)
 
+    def test_to_latex_multicolumnrow(self):
+        df = pd.DataFrame({
+                    ('c1',0):dict((x,x) for x in range(5)),
+                    ('c1',1):dict((x,x+5) for x in range(5)),
+                    ('c2',0):dict((x,x) for x in range(5)),
+                    ('c2',1):dict((x,x+5) for x in range(5)),
+                    ('c3',0):dict((x,x) for x in range(5))
+                    })
+        result = df.to_latex(multicolumn=True)
+        expected = r"""\begin{tabular}{lrrrrr}
+\toprule
+{} & \multicolumn{2}{l}{c1} & \multicolumn{2}{l}{c2} & c3 \\
+{} &  0 &  1 &  0 &  1 &  0 \\
+\midrule
+0 &  0 &  5 &  0 &  5 &  0 \\
+1 &  1 &  6 &  1 &  6 &  1 \\
+2 &  2 &  7 &  2 &  7 &  2 \\
+3 &  3 &  8 &  3 &  8 &  3 \\
+4 &  4 &  9 &  4 &  9 &  4 \\
+\bottomrule
+\end{tabular}
+"""
+        self.assertEqual(result, expected)
+
+        result = df.T.to_latex(multirow=True)
+        expected = r"""\begin{tabular}{llrrrrr}
+\toprule
+   &   &  0 &  1 &  2 &  3 &  4 \\
+\midrule
+\multirow{2}{*}{c1} & 0 &  0 &  1 &  2 &  3 &  4 \\
+   & 1 &  5 &  6 &  7 &  8 &  9 \\
+\cline{1-7}
+\multirow{2}{*}{c2} & 0 &  0 &  1 &  2 &  3 &  4 \\
+   & 1 &  5 &  6 &  7 &  8 &  9 \\
+\cline{1-7}
```

Comment:
Another question for the default: would we rather want to have it top-aligned instead of center? (here it may look better centered, but once you don't have exactly the same number of rows in each subpart, then I think top-aligned will look better)